cmd/cloud-api-adaptor/main.go

```diff
@@ -36,7 +36,7 @@ type networkConfig struct {
 	VXLANMinID    int
 }
 
-var azurecfg azure.Config
+var vspherecfg vsphere.Config
```

Comment:
vSphere has been migrated to the new code layout. Could you remove this line?

Reply:
Thank you, might have been missed during rebase. Fixed